src/lights/HemisphereLight.js

```diff
@@ -2,36 +2,34 @@ import { Light } from './Light.js';
 import { Color } from '../math/Color.js';
 import { Object3D } from '../core/Object3D.js';
 
-function HemisphereLight( skyColor, groundColor, intensity ) {
+class HemisphereLight extends Light {
 
-	Light.call( this, skyColor, intensity );
+	constructor( skyColor, groundColor, intensity ) {
 
-	this.type = 'HemisphereLight';
+		super( skyColor, intensity );
 
-	this.position.copy( Object3D.DefaultUp );
-	this.updateMatrix();
+		this.type = 'HemisphereLight';
+		Object.defineProperty( this, 'isHemisphereLight', { value: true } );
 
-	this.groundColor = new Color( groundColor );
+		this.castShadow = undefined;
```

Comment:
And this one. Where did this come from?

Reply:
[Same story with this one](https://github.com/mrdoob/three.js/commit/bd4f0c2d07c9e9c30ec897b6faa69a7a835762b4#diff-f61670c13d171254bc9a629aabde40bb4bf70b06753f58ceeb204131c955e415)